index.tt

```diff
@@ -6,152 +6,54 @@
-      provides atomic upgrades and rollbacks, side-by-side installation
-      of multiple versions of a package, multi-user package management
-      and easy setup of build environments.
-      </p>
-    
-      <div class="learn-more">
-        <a class="btn btn-success" href="[%root%]nix/">
-          <i class="fa fa-chevron-right"></i> Learn more about Nix
-        </a>
-      </div>
-    </div>
-    <div class="intro-nixos">
-      <h2>What is NixOS?</h2>
-    
-      <p>NixOS is a Linux distribution with a unique approach
-      to package and configuration management. Built on top of the
-      Nix package manager, it is completely declarative, makes
-      upgrading systems reliable, and has
-      <a href="[%root%]nixos/about.html">many other advantages</a>.</p>
-    
-      <div class="learn-more">
-        <a class="btn btn-success" href="[%root%]nixos/">
-          <i class="fa fa-chevron-right"></i> Learn more about NixOS
-        </a>
-      </div>
-    </div>
-  </div></div>
-  
-  <hr />
-  
-  <h2>More about NixOS</h2>
-  
-  <div class="row-fluid more-about">
-    <div class="span4">
-      <h3>Nixpkgs</h3>
-      <p>
-        The Nix Packages collection (Nixpkgs) is the ever growing
-        packages collection for the Nix package manager, released
-        under a permissive MIT/X11 license.
-      </p>
-      <a class="learn-more" href="[%root%]nixpkgs">
-        More about Nixpkgs
-        <i class="fa fa-angle-right"></i>
-      </a>
-    </div>
-    <div class="span4">
-      <h3>NixOps</h3>
-      <p>
-        NixOps is a tool for deploying sets of NixOS Linux machines,
-        bare-metal or to the cloud. It extends NixOS’s declarative
-        approach to system configuration management to networks and
-        provisioning.
-      </p>
-      <a class="learn-more" href="https://github.com/NixOS/nixops">
-        More about NixOps
-        <i class="fa fa-angle-right"></i>
-      </a>
-    </div>
-    <div class="span4 and-more">
-      <h3>About the community</h3>
-      <p>
-        Sometimes, it helps to talk with someone.  Our community
-        is there whether you want to help or get support.
-      </p>
-      <ul>
-        <li>
-          <a href="[%root%]nixos/community.html">
-            Get in touch
-            <i class="fa fa-angle-right"></i>
-          </a>
-        </li>
-        <li>
-          <a href="[%root%]learn.html">
-            Learning resources
-            <i class="fa fa-angle-right"></i>
-          </a>
-        </li>
-      </ul>
-    </div>
-  </div>
-</div>
-
-<hr />
-
 <div class="row-fluid">
+
   <div class="span6">
-    <h2>
-      <a href="/news-rss.xml">
-        <img src="/logo/rss.png"/>
-      </a>
-      News
-    </h2>
-    [% INSERT "latest-news.xhtml" %]
-    <p><a href="news.html" class="btn btn-mini">More »</a></p>
+    <h1>Reproducible builds and deployments.</h1>
+    <p class="lead">
+      NixOS is an open source set of tools that help you package and deliver
```

Comment:
i don't know if i should already comment this, but here are my spontaneous thoughts

>NixOS is an open source set of tools

that's not a very specific description. you could say that about many projects that are totally unrelated. i think "Linux distribution" should be in there. we are already way ahead of what people know, give them something they can grasp

Reply:
:+1: agree I'm so not happy with this description. But at least we have a titie :)

Reply:
yes, that's something :sparkles: